Dockerfile

```diff
@@ -39,6 +39,7 @@ RUN if [ $STATIC_BUILD = "TRUE" ] ; \
   then SECRET_KEY=dummy_value poetry run python manage.py distill-local build --traceback --force ; \
 fi
 
-# Run web server through custom manager
-ENTRYPOINT ["poetry", "run", "python", "manage.py"]
-CMD ["run"]
+CMD ["poetry", "run", "gunicorn", "--preload", "-b", "0.0.0.0:8000", \
```

Comment:
Would it be more convenient to have `poetry run` be the entrypoint?

Reply:
That's a good call, updated in force push.